tests/comfy_cli/command/test_run.py

```
@@ -185,15 +185,26 @@ def test_queue_does_not_send_x_api_key_header(self, workflow):
             ex.queue()
         req = mock_open.call_args[0][0]
         body = json.loads(req.data)
-        assert "extra_data" not in body
-        assert body == {"prompt": workflow, "client_id": ex.client_id}
+        assert body == {
+            "prompt": workflow,
+            "client_id": ex.client_id,
+            "extra_data": {"comfy_usage_source": "comfy-cli"},
+        }
+
+    def test_queue_sends_usage_source_header(self, workflow):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)